CHANGELOG.md

```diff
@@ -3,6 +3,7 @@ All notable changes to this project will be documented in this file.
 This project uses the changelog in accordance with [keepchangelog](http://keepachangelog.com/). Please use this to write notable changes, which is not the same as git commit log...
 
 ## [unreleased][unreleased]
+ - Changed `hf mf sim` - reduce 50ms threshold to 4ms for reset to idle #1974 (@net147)
```

Comment:
the comment is wrong,  it changes from 6ms to 4ms.

Reply:
Done.